adafruit_debouncer.py

```diff
@@ -106,6 +106,14 @@ def update(self, new_state: Optional[int] = None) -> None:
                     )
                     self._state_changed_ticks = now_ticks
 
+    def force_set(self, state: bool)->None:
+        """Manually set debounced value and trigger .rose or .fell
+        :param bool state: Value you want to manually set.
+        """
+        if self._get_state(_DEBOUNCED_STATE) != state:
+            self._set_state(_CHANGED_STATE)
+            self._toggle_state(_DEBOUNCED_STATE)
+
```

Comment:
If `force_state` is going to behave in the same way as when an event organically happens, does it need to also set any of `_last_bounce_ticks`, `_last_duration_ticks` or `_state_changed_ticks`? Would it be tidier to design this so it can call `update()` and let all of the state change logic be handled consistently whether the state change event is being induced or is happening from the environment?

Reply:
Looking back on this, I think somewhere in the middle is a good balance - we should change it to set all the aforementioned attributes, but explicitly NOT call `update()` since there's utility in using this new `force_set()` to change without calling what could be a function (self.function()`) that might change other undesirable things as well.  So `force_set()` and `update()` would pull from this third new bit of functionality.